lib/src/components/NxSearchTransferList/NxSearchTransferList.tsx

```diff
@@ -40,8 +40,8 @@ export default function NxSearchTransferList<T extends string | number>(props: P
       addedCount = addedItems.length,
       addedItemsCountFormatter = addedItemsCountFormatterProp || defaultAddedItemsCountFormatter;
 
-  function onRemoveAll(idsToRemove: Set<T>) {
-    const newAddedItems = reject(({ id }) => idsToRemove.has(id), addedItems);
+  function onRemoveAll(idsToRemove: T[]) {
+    const newAddedItems = reject(({ id }) => idsToRemove.includes(id), addedItems);
```

Comment:
I wonder if it's worth it to collect the ids into a Set in a local variable first so that each lookup is O(1) not O(n)

Reply:
> I wonder if it's worth it to collect the ids into a Set in a local variable first so that each lookup is O(1) not O(n)

Thanks, I think it's worth it: 774e9702b6a1123bac0705354974edd6087775a5